rules/S6877/java/metadata.json

```
@@ -0,0 +1,24 @@
+{
+  "title": "SequencedCollection reversed view should be used instead of Collections.reverse for read-only lists",
```

We generally put in quotes methods, classes and so on inside the metadata title
```suggestion
  "title": "\"SequencedCollection\" reversed view should be used instead of \"Collections.reverse\" for read-only lists",
```